src/edu/csus/ecs/pc2/clics/API202306/CLICSProblemScore.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.HashMap;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.model.ContestTime;
+import edu.csus.ecs.pc2.core.standings.ProblemSummaryInfo;
+
+/**
+ * CLICS Score (for a problem)
+ * Contains information about the score a team received for a single problem
+ * 
+ * @author John Buck
+ *
+ */
+
+public class CLICSProblemScore {
+
+    @JsonProperty
+    private String problem_id;
+
+    @JsonProperty
+    private int num_judged;
+
+    @JsonProperty
+    private int num_pending;
+
+    @JsonProperty
+    private boolean solved;
+
+// Not needed for pass-fail contest
+//  @JsonProperty
+//  private double score;
+
+    @JsonProperty
+    private String time;
+
+    /**
+     * Fill in API problem score information properties (for scoreboard endpoint)
+     * 
+     * @param probEleToShort hashmap for mapping problem elementid to shortname
+     * @param versionInfo
+     */
+    public CLICSProblemScore(HashMap<String, String> probEleToShort, ProblemSummaryInfo psi) {
+        num_judged = Utilities.nullSafeToInt(psi.getAttempts(), 0);
```

Comment:
The call to `psi.getAttempts()` worries me.  That "getter" returns the value of the private field `attempts` in the PSI.  Since that field is private, the only way it can have a reasonable (i.e. non-zero) value is if some code somewhere calls `PSI.setAttempts()`.  However, there's not a single reference to `PSI.setAttempts()` anywhere in the entire PC2 project.  So how does the value returned by `getAttempts()` represent the value which should be stored in`num_judged`?

It also bothers me that the field `attempts` in PSI is a _string_... but that would be workable _if_ someone ever assigned a value to _attempts_.

Reply:
`ProblemSummaryInfo` is an `javax.xml` type object.  It is used to deserialize XML.  Recall the DSA (and NSA) both create an XML representation of the scoreboard.   This class deserializes a piece of that XML scoreboard into an object for use in such things as, well, this.  As such, the setter methods will not be called when it is used to deserialize XML, but, the fields do get set.